fluent-langneg/test/locale_test.js

```diff
@@ -51,11 +51,11 @@ suite('Parses simple locales', () => {
       variant: 'mac'
     }));
 
-    assert.ok(isLocaleEqual('lij-Arab-FA-linux', {
+    assert.ok(isLocaleEqual('lij-Arab-FA-lin', {
```

Comment:
We only allow the variant code to be 3 chars long. If we want to relax it, I'll be happy to use `-linux`.